src/Common/ThreadFuzzer.cpp

```diff
@@ -172,6 +176,8 @@ void ThreadFuzzer::stop()
 
 void ThreadFuzzer::start()
 {
+    if (!instance().isEffective())
```

Comment:
start() checks for isEffective, and isEffective checks for isStarted(). Result: Impossible to start it. So fast :smile:

Reply:
Some consequences:
* `SYSTEM START THREAD FUZZER` is useless. Once the fuzzer is stopped it can't start.
* If you don't use `THREAD_FUZZER_CPU_TIME_PERIOD_US` sleep faults won't work because isEffective will return false.

Reply:
https://github.com/ClickHouse/ClickHouse/pull/63775

Reply:
>start() checks for isEffective, and isEffective checks for isStarted(). Result: Impossible to start it. So fast 😄

What a shame... But it should affect only `SYSTEM START THREAD FUZZER`, while the overhead was there all the time.